logic/transactionPool.js

```diff
@@ -101,7 +101,9 @@ TransactionPool.prototype.transactionInPool = function (id) {
 		self.bundled.index[id],
 		self.queued.index[id],
 		self.multisignature.index[id]
-	].filter(Boolean).length > 0;
+	].filter(function (index) {
+		return typeof(index) === 'number';
```

Comment:
@4miners I'd disagree about the clearness of doing `index !== undefined`.
While the result would be the same (Starting from es5 since before undefined used to be mutable) it does not give any hint on what index *should* be. 

Keeping it as I proposed will clearly give code-reader a clue of what `self.bundled.*[id]` should be => a number.